surveys/survey-validator/bin/survey_validator.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:io';
+
+import 'package:survey_validator/survey_validator.dart' as survey_validator;
+
+void main(List<String> arguments) {
+  final contextualSurveyFile = File('../contextual-survey-metadata.json');
+
+  survey_validator.checkJson(contextualSurveyFile);
+}
```

Comment:
@parlough does this look okay for getting the file path to the metadata json file?

Reply:
Yeah that's a good point, if we expand to additional files that need to be validated I will consider this, thank you! :)